sklearn/tests/test_cross_validation.py

```diff
@@ -289,12 +291,10 @@ def test_shuffle_kfold():
 
     all_folds = None
     for train, test in kf:
-        sorted_array = np.arange(100)
-        assert_true(np.any(sorted_array != ind[train]))
-        sorted_array = np.arange(101, 200)
-        assert_true(np.any(sorted_array != ind[train]))
-        sorted_array = np.arange(201, 300)
-        assert_true(np.any(sorted_array != ind[train]))
+        assert_true(np.any(np.arange(100) != ind[test]))
+        assert_true(np.any(np.arange(100, 200) != ind[test]))
```

Comment:
```python
np.arange(100) = [0, ..., 99] (100 elements)
np.arange(100, 200) = [100, ..., 199] (100 elements)
np.arange(101, 200) = [101, ..., 199] (99 elements)
```
The test was asserting the arrays are different because of different sizes, and not because of the shuffling.